pages/api/id/[form]/owners.ts

```diff
@@ -80,13 +80,12 @@ export async function activateOrDeactivateFormOwners(
 ): Promise<void> {
   try {
     //Extracting req body
-    const requestBody = req.body ? req.body : undefined;
+    const { email, active } = req.body;
```

Comment:
This line will throw an error if `req.body` was undefined. To prevent this behaviour it's safe to set a fallback value for `body`.   `const { email, active } = req.body || {}`.